dm/checker/checker.go

```diff
@@ -330,6 +332,45 @@ func (c *Checker) Init(ctx context.Context) (err error) {
 		}
 	}
 
+	if instance.cfg.Mode != config.ModeIncrement && instance.cfg.LoaderConfig.ImportMode == config.LoadModePhysical {
```

Comment:
Do we need some checks for tidb backend?

Reply:
the checks in this PR is only for local backend because they are related to storage level. I'll add more later

Reply:
Does DM support local backend now? 🤔

Reply:
https://github.com/pingcap/tiflow/pull/7448